lib/foreman_custom_tab/engine.rb

```diff
@@ -9,29 +9,29 @@ class Engine < ::Rails::Engine
     config.autoload_paths += Dir["#{config.root}/app/helpers/concerns"]
     config.autoload_paths += Dir["#{config.root}/app/models/concerns"]
     config.autoload_paths += Dir["#{config.root}/app/overrides"]
-  
-    
-    initializer 'foreman_custom_tab.configure_assets', :group => :assets do
-      SETTINGS[:foreman_custom_tab] = { assets: { precompile: assets_to_precompile } }
-    end
     
+    # requires_foreman version as per
+    # http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Requiring-Foreman-version
+
+    # Adding permission to controller action
+    # http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Adding-permission
     initializer('foreman_custom_tab.register_plugin', :before => :finisher_hook) do
       Foreman::Plugin.register :foreman_custom_tab do
-        requires_foreman '>= 1.7'
+        requires_foreman '>= 1.8'
         
         security_block :foreman_custom_tab do
           permission :view_hosts,
-              { :hosts => [:summary] },
-              :resource_type => 'Host'
+              { 
+                :controller_name => [:show]
```

Comment:
This does not seem right. Have you tested it? It does not mention our new controller action custom_tab here. You used show.

Reply:
I think we tried this anyways and it failed. We copied the other syntax from another plugin